src/main/java/seedu/manager/command/AddCommand.java

```diff
@@ -6,10 +6,12 @@
  */
 public class AddCommand extends Command {
     public static final String COMMAND_WORD = "add";
-    private static final String ADD_MESSAGE = "Event added successfully";
+    private static final String ADD_E_MESSAGE = "Event added successfully";
+    private static final String ADD_P_MESSAGE = "Participant added successfully";
```

Comment:
I think these two constants could be named "ADD_EVENT_MESSAGE" and "ADD_PARTICIPANT_MESSAGE" to make their purpose clearer?